baggage/utils.py

```diff
@@ -0,0 +1,51 @@
+import math
+import itertools
+from baggage.models import Room, Bag, BAG_ADDED
+
+
+def calculate_distance(name, ini_x, ini_y, end_x, end_y):
+    return ((name, end_x, end_y), math.sqrt(pow(abs(end_x - ini_x), 2) + pow(abs(end_y - ini_y), 2)))
+
+
+def nearest_available(x, y):
+    return ''
+
+
+def get_positions(size_x, size_y, door_x, door_y):
+    return list(itertools.product(list(map(str, range(0, size_x))), list(map(str, range(0, size_y)))))
```

Comment:
Consider using intermediate variables or a better indentation to improve readability